src/cavendish_particle_tracks/_magnification_dialog.py

```diff
@@ -113,22 +143,22 @@ def _setup_dropdown_fiducials_combobox(self, back=False):
 
     def _on_click_add_coords_f1(self) -> None:
         """Add first front fiducial"""
-        self.f1.name = self.cbf1.currentText()
+        self.f1.name = self.cmb_front1.currentText()
         self.f1.x, self.f1.y = self._add_coords(0)
 
     def _on_click_add_coords_f2(self) -> None:
         """Add second front fiducial"""
-        self.f2.name = self.cbf2.currentText()
+        self.f2.name = self.cmb_front2.currentText()
```

Comment:
I've already forgotten what `cb` and `cmb` are 🙃 . Combo box?

Reply:
changed `cmb` to `combobox`